Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -245,12 +245,7 @@ public class ProgramBuilder {
     
     /// Type information access.
     public func type(of v: Variable) -> Type {
-        let runtimeType = program.runtimeType(of: v)
-        if runtimeType == .unknown {
-            return interpreter?.type(of: v) ?? .unknown
-        } else {
-            return runtimeType
-        }
+        return interpreter?.type(of: v) ?? .unknown
```

Comment:
Nice. With that change, you can probably also remove this code: https://github.com/googleprojectzero/fuzzilli/blob/66de56c39e298e53f2a40c86f7b4607a56a37bc6/Sources/Fuzzilli/Core/ProgramBuilder.swift#L345 ?

Reply:
hmm and where will runtime types be stored?